pegasus/sites.v3/hourofcode.com/public/highlights.haml

```diff
@@ -0,0 +1,36 @@
+---
+title: Highlights
+layout: wide
+---
+
+%link{href: "/css/highlights.css", rel: "stylesheet"}
+
+%h1 Hour of Code Highlights
+%h3 Some of our favorite moments of the Hour of Code campaign.
+
+%div.highlights-content
+  - tag = 'highlight'
+  -DB[:cdo_news_items].where(tag_s:tag).each do |item|
+    .highlight-item
+      %img.higlight-img{src: "#{item[:image_url_s]}"}
+      .highlight-caption
+        - if item[:short_heading_s]
+          %h4.caption-headline= item[:short_heading_s]
```

Comment:
I wonder if you could do

    .highlight-caption
       %h4.caption-headline= item[:short_heading_s] unless item[:short_heading_s]nil?